scalar_serializer.go

```diff
@@ -61,48 +61,48 @@ func (scalarSerializer *scalarSerializer) scalar(ionValue interface{}) error {
 	}
 
 	tq, representation, err :=
-		scalarSerializer.scalarOrNullSplitParts(ionType, ionHashValue.IsNull(), scalarBytes)
+		ss.scalarOrNullSplitParts(ionType, ionHashValue.IsNull(), scalarBytes)
```

Comment:
Previously this was split over two lines because it was too long but now you've shortened it. Can you check to see if this line (and others like it) are now a suitable length for a single line?

Reply:
Moved to one line